Makefile

```diff
@@ -74,11 +74,11 @@ build: generate fmt vet golangci-lint kube-linter ## Build manager binary.
 run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
-docker-build: generate fmt vet golangci-lint kube-linter ## Build docker image with the manager.
-	docker build -t ${IMG} .
+operator-build: generate fmt vet golangci-lint kube-linter ## Build docker image with the manager.
```

Comment:
Why do we need to rename it?

Reply:
Renaming it to make the command more generic. 
operator-build and operator-push will not make assumption of the tool being used.